src/Apps/Search/Components/SearchResultsSkeleton/index.tsx

```diff
@@ -17,17 +17,17 @@ export const SearchResultsSkeleton: React.FC<any> = props => {
             <Box width={["100%", "75%"]}>
               <Grid fluid>
                 <Row>
-                  <Col sm={4} pr={1}>
+                  <Col xl="4" lg="4" md="6" sm="6" xs="6">
```

Comment:
The grid props work similarly to the responsive props in styled-system, where it will take the last lowest value and apply it upwards. So this can be simplified to: 
```
<Col xs='6' md='4' />
```

Reply:
Ahh, great! Thank you!!